app/components/Channels/ChannelCreateForm.js

```diff
@@ -78,6 +130,31 @@ class ChannelCreateForm extends React.Component {
     }
   }
 
+  handleSubmit = values => {
+    const { onSubmit } = this.props
+    const { step } = this.state
+
+    if (step === 'form') {
+      return this.setState({ step: 'summary' })
+    } else {
```

Comment:
Else could be omitted, but I guess it's style pref, so up to you

Reply:
Updated.